### DISCUSSION TITLE
R not responding after opening big object

### DISCUSSION THREAD

**Discussion by D-Obwegs** [posit-dev/positron#5333] [Q&A]
Hi,

I am a biologist/data scientist and am mostly working on single-cell RNA sequencing datasets, which can get huge. Recently, I tried switching to Positron instead of Rstudio, since I am also using Python.
Python is working flawlessly, but whenever I try to open a bigger object in R, R becomes unresponsive. I can still navigate the IDE and run python code, but if I try to run R code, the interrupting option appears but the code is not executed. No error message or anything else appears and I couldn't find any log files. The object also doesn't appear in the variables window. It also doesn't matter if I try to load a Seurat (the package I use for scRNA-seq analysis) object or if I try to open the matrix directly using the Matrix package.

Positron: 2024.11.0-140
R: 4.4.2

I hope I could make my problem somewhat clear. Any help would be appreciated. Thanks!

**Comment by jmcphers** [comment]
This is a known issue and currently tracked in https://github.com/posit-dev/positron/issues/3628; feel free to add any notes/details there. Thanks for reporting!
